common/src/main/java/tachyon/security/PlainSaslServer.java

```diff
@@ -39,7 +38,7 @@
  * 3.Write a JCA provider that registers the factory
  */
 public class PlainSaslServer implements SaslServer {
-  private String mAuthcid;
+  private String mAuthzid;
```

Comment:
I agree username should be authorization ID.  But it is not `mAuthcid`.

Actually, `mAuthcid` is authentication ID, and `mAuthzid` is authorization ID. Original code logic save the authentication ID(`mAuthcid`) as username, which is not expected beharior. So it needs to be changed here.